config/config.go

```diff
@@ -52,6 +52,9 @@ const (
 	SettingDbUsername = "mongo_username"
 	SettingDbPassword = "mongo_password"
 
+	SettingLimitSessionsPerUser        = "limit_sessions_per_user"
+	SettingLimitSessionsPerUserDefault = 10
```

Comment:
do we need an entry here: https://github.com/mendersoftware/useradm/blob/master/config.yaml ?